packages/scatterplot/stories/ScatterPlotCanvas.stories.js

```diff
@@ -125,6 +125,11 @@ const commonProps = {
     height: 500,
     margin: { top: 20, right: 20, bottom: 80, left: 80 },
     symbolSize: 10,
+    symbol: (ctx, point) => {
```

Comment:
It could be useful to have a demo of custom symbol, but probably not by default IMO

Reply:
True. I removed it.

Do you want me to create another story or to add it to an existing one ?

Reply:
That would be nice!